packages/core/directives/map.ts

```diff
@@ -97,6 +97,11 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
    * are enforced.
    */
   @Input() maxZoom: number;
+  
```

Comment:
tslint errors me out on 

> ERROR: packages/core/directives/map.ts:100:1 - trailing whitespace
> ERROR: packages/core/directives/map.ts:102:1 - asterisks in jsdoc must be aligned
> ERROR: packages/core/directives/map.ts:103:1 - asterisks in jsdoc must be aligned

Reply:
This is also what fails the build, if you noticed.

Reply:
I didn't notice. Will check for future pull requests.